src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -866,10 +866,10 @@ def fix_check_failure_string(failure_string):
   # Cover example like "CHECK_EQ( (unsigned)ptr[0],1u) failed: 25 vs. 1".
   # This only happens on Android, where we cannot strip the
   # CHECK_FAILURE_PATTERN, so we looked for "failed:" as preceding string.
-  failure_string = re.sub(r'(?<=failed): .*vs\..*$', r'', failure_string)
+  failure_string = re.sub(r'(?<=failed): .* vs\. .*$', r'', failure_string)
```

Comment:
Are those spaces? Should we use `\s` instead?

Reply:
May be even `\s+`, just in case?

Reply:
\s seems good, if there are more space, they just get ignored. our condition needs to do a minimal check.